docs/userguide.rst

```diff
@@ -82,7 +82,8 @@ These user guides provide additional details about specific features in UXarray.
 `Compatibility with HoloViz Tools <user-guide/holoviz.ipynb>`_
  Use UXarray with HoloViz tools
 
-
+`Grid Function: from_file and Shapefile Remapping<user-guide/from_file.ipynb>`_
+ Load unstructured grids using the Grid from_file function
```

Comment:
This looks to have rendered incorrectly on the docs.

![image](https://github.com/user-attachments/assets/3dad29ec-d9cd-41e3-b651-4b52251cd757)